README.md

```diff
@@ -55,7 +55,7 @@ Skrop provides a set of filters, which you can use within the routes:
 * **crop(width, height, type)** — crops the image to have the specified width and height the type can be "north", "south", "east" and "west"
 * **cropByHeight(height, type)** — crops the image to have the specified height
 * **cropByWidth(width, type)** — crops the image to have the specified width
-* **resize(width, height)** — resizes an image without preserving its aspect ratio
+* **resize(width, height, <keep proportions?>)** — resizes an image. Third parameter is optional: true to keep proportions, false to ignore proportions (default to true)
```

Comment:
I would rename proportions to aspect ratio... it's a more common term for images.